tests/utils/mod.rs

```diff
@@ -7,18 +6,14 @@ const SERVER_PATH: &str = "/gtfs_rt";
 
 pub fn make_test_server() -> actix_web::test::TestServer {
     LOGGER_INIT.call_once(|| env_logger::init());
-    let begin = NaiveDate::from_ymd(2018, 12, 15);
-    let period = transpo_rt::context::Period {
-        begin: begin.clone(),
-        end: begin.succ(),
-    };
+    let horizon = chrono::Duration::days(2);
```

Comment:
I don't really see how this can work, for the test data we want to fix a requested date no ?

Reply:
Better now